joomla/joomla-cms
# Use the new native title metadata in installed languages view

### Summary of Changes

Use the new native name metadata in installed languages view

![image](https://cloud.githubusercontent.com/assets/9630530/19836618/f096ef78-9e9d-11e6-9c41-cae3db3537df.png)
### Testing Instructions
1. Use latest staging
2. Apply patch
3. Go to Extensions >Languages > Installed
4. Check there is a new column (Native Title) with the info
5. Test site and admin languages.

The code was also optimized to use the new JLanguageHelper::getInstalledLanguages method.
### Documentation Changes Required

None.

@infograf768 please check this.
